x-pack/plugins/stack_connectors/common/inference/schema.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const ConfigSchema = schema.object({
+  provider: schema.string(),
+  taskType: schema.string(),
```

Comment:
This is the intensional behavior. When the new provider will be implemented by the Inference Endpoint team, it will automatically become available in Kibana within the getProviders API (which is hardcoded for 8.16, but will be available in 8.17). That will bring the flexibility implementing provider integration at the single place - Elasticsearch. And Kibana will get it.

Reply:
Okay, I wasn't aware this was planned already. Then it makes sense 👍